eng/CodeCoverage.props

```diff
@@ -0,0 +1,10 @@
+<Project>
```

Comment:
Could we have some auto-import in Directory.build.props or target with some condition to filter out vstest like projects(e.g. `UseVsTestRunner`).

Reply:
I was wondering that this props make sense only in the context of test runner empowered tests and to avoid to have mixed props or possible collision till we're mixed runner for arcade, but if you want we can  have shared, looks to me for now more "clear" and explicit that it's "for this project" only.

Reply:
We currently default to saying all tests use vstest runner and we specify on case by case when we want to use TA runner (see https://github.com/microsoft/testfx/blob/main/Directory.Build.Local.props#L65-L66 for default config). If you import the code coverage props in the directory build local targets with the condition on vstest runner it should work, or am I missing something?